src/Environment/Config.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CloudPatches\Environment;
+
+use Magento\CloudPatches\Filesystem\FileSystemException;
+
+/**
+ * Environment configuration.
+ */
+class Config
+{
+    /**
+     * Variable to define a Cloud environment.
+     */
+    const ENV_VAR_CLOUD = 'MAGENTO_CLOUD_PROJECT';
```

Comment:
This is not needed since you have separate entry point for Cloud

Reply:
The reason is that Status command is common for ece-patches and magento-patches. This command shows a table with patch type.  I believed that is good to show all Clod patches as required even if the customer runs './magento-patches status' on the Cloud environment. Otherway cloud patches will display as optional. Does it make sense?